packages/datatrak-web/src/features/MobileAppPrompt.tsx

```diff
@@ -0,0 +1,184 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { Link, SwipeableDrawer, Typography } from '@material-ui/core';
+import React, { useState } from 'react';
+import { useIsMobile } from '../utils';
+import styled from 'styled-components';
+import { Button } from '@tupaia/ui-components';
+
+const Container = styled.div`
+  padding: 2.2rem 1.25rem 3.4rem 1.25rem;
+`;
+
+const Title = styled(Typography).attrs({
+  variant: 'h2',
+})`
+  font-size: 1.125rem;
+  text-align: center;
+  font-weight: ${({ theme }) => theme.typography.fontWeightMedium};
+`;
+
+const LeftColumn = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const Option = styled.div`
+  display: flex;
+  align-items: center;
+  margin-top: 1rem;
+  justify-content: space-between;
+`;
+
+const OptionIcon = styled.div`
+  border: 1px solid ${({ theme }) => theme.palette.divider};
+  width: 2.8rem;
+  height: 2.8rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0.3rem;
+  border-radius: 4px;
+  margin: 0 0.625rem;
+  img {
+    height: 100%;
+    width: auto;
+  }
+`;
+
+const OptionText = styled(Typography)`
+  font-weight: ${({ theme }) => theme.typography.fontWeightMedium};
+`;
+
+const OptionButton = styled(Button)`
+  border-radius: 2rem;
+  width: 7rem;
+  font-weight: 400;
+`;
+
+const BROWSERS = {
+  CHROME: 'Chrome',
+  FIREFOX: 'Firefox',
+  SAFARI: 'Safari',
+  OPERA: 'Opera',
+  EDGE: 'Edge',
+};
+
+const BROWSER_ICONS = {
+  [BROWSERS.CHROME]: '/chrome-icon.png',
+  [BROWSERS.FIREFOX]: '/firefox-icon.png',
+  [BROWSERS.SAFARI]: '/safari-icon.png',
+  [BROWSERS.OPERA]: '/opera-icon.png',
+  [BROWSERS.EDGE]: '/edge-icon.png',
+};
+
+const APP_URL = {
+  ANDROID: 'https://play.google.com/store/apps/details?id=com.tupaiameditrak',
+  IOS: 'https://itunes.apple.com/us/app/tupaia-meditrak/id1245053537',
+};
+
+export const MobileAppPrompt = () => {
```

Comment:
thanks for doing this - It's a bit above and beyond the scope of the original ticket so appreciated